### PR TITLE
Modal: should auto focus modal after open

### LINKED ISSUE #610: Modal: should auto focus modal right after open
https://drive.google.com/file/d/1qnf4uPD2gGs6aa3R0lsAOb2geRfgGJVX/view

### PR DESCRIPTION
**Pull request by rosslo**

### CODE REVIEW

File: src/Display/Modal/Modal.tsx [hunk 1]
```diff
@@ -73,7 +74,7 @@ export const Modal: FC<Props> = ({
 
     if (isVisible) {
       // On modal open
-      scrollableTarget.focus();
```
Comment (rosslo) [inline src/Display/Modal/Modal.tsx:76]: Have tried `scrollableTarget.tabindex = 0` but it didn't work. So I just revert the change. https://github.com/glints-dev/glints-aries/pull/608/files#diff-88e4bf70b17929c5c287f91a7c665ed684dcf98c47a7e2b72c3ea003a82b9e81L65

**Review by ritikamotwani — APPROVED** [review]
